mkdocs/docs/HPC/index.md

```diff
@@ -1,10 +1,32 @@
-# Welcome
+---
+hide: "toc"
+---
+# HPC Documentation
+
+---
+
+
+![image](img/ch0-hpc.png){: align=left style="height:383px;width:378px;border-radius:6px" }
+<!-- 1150 1134 -->
 
 Welcome to the {{ hpcinfra }} documentation!
 
 Use the menu on the left to navigate, or use the search box on the top right.
 
-The current view of the documentation is intended for people using **{{ OS }}** as operating system.
-To switch to different operating system (Windows, macOS, Linux), click [here](../#force_select_OS).
+You are viewing documentation intended for people using **{{ OS }}**. To switch to different operating system (Windows, macOS, Linux), click [HERE](../#force_select_OS).
+
+**Quick links**
+
+- [Introduction to HPC](introduction.md)
+- [Getting Started]() <!-- TODO: add link to getting started -->
+{%- if site == 'Gent' %}
+- [Linux Tutorial](./only/gent/linux-tutorial/index.md)
+- [Hardware overview](https://www.ugent.be/hpc/en/infrastructure)
+{%- endif %}
+- [FAQ](FAQ.md) - [Troubleshooting](troubleshooting.md) - [Best practices](./best_practices.md)
```

Comment:
Maybe better:

```
- Getting Started - Getting Access
- Recording of HPC-UGent intro - Linux tutorial
- HPC-UGent hardware overview
- FAQ - Troubleshooting - Best practices
- Overview of available software
```

last page is not there yet, but room should be left to link to it once @milachae's work on generating an overview of available software has progressed